spec/services/spree/solidus_six_saferpay/initialize_transaction_spec.rb

```diff
@@ -3,34 +3,32 @@
 module Spree
   module SolidusSixSaferpay
     RSpec.describe InitializeTransaction do
+      subject(:initialize_transaction) { described_class.new(order, payment_method) }
 
       let(:order) { create(:order) }
       let(:payment_method) { create(:saferpay_payment_method) }
 
-      subject { described_class.new(order, payment_method) }
-
-
       describe '#gateway' do
         it_behaves_like "it uses the transaction gateway"
       end
 
       describe '#call' do
-        let(:token) {  '234uhfh78234hlasdfh8234e1234' }
-        let(:expiration) { '2015-01-30T12:45:22.258+01:00' }
-        let(:redirect_url) { '/saferpay/redirect/url' }
-
         # https://saferpay.github.io/jsonapi/#Payment_v1_Transaction_Initialize
         let(:api_response) do
           SixSaferpay::SixTransaction::InitializeResponse.new(
             response_header: SixSaferpay::ResponseHeader.new(request_id: 'test', spec_version: 'test'),
-            token: token,
-            expiration: expiration,
-            liability_shift: nil, # this is empty because we don't submit PaymentMeans on initialize already
-            redirect_required: true, 
-            redirect: SixSaferpay::Redirect.new(redirect_url: redirect_url, payment_means_required: true) # since we don't provide PaymentMeans on initialize, we must always redirect
+            token: '234uhfh78234hlasdfh8234e1234',
+            expiration: '2015-01-30T12:45:22.258+01:00',
+
+            # this is empty because we don't submit PaymentMeans on initialize already
+            liability_shift: nil,
+
+            # since we don't provide PaymentMeans on initialize, we must always redirect
```

Comment:
Change to `it must always be redirected, since PaymentMeans are not provided on initialize`.